src/db.py

```diff
@@ -0,0 +1,132 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, Float, String, MetaData
+import sqlalchemy as sql
+import logging
+import os
+import pandas as pd
+from scipy.stats import mode
+import argparse
+
+Base = declarative_base()
+
+# set up looging config
```

Comment:
Is the logging config necessary again given you have a whole config file for it?